OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -20,12 +21,15 @@ public class ScrollItemWidget : ButtonWidget
 		public readonly bool EnableChildMouseOver = false;
 		public string ItemKey;
 
+		readonly CachedTransform<(bool, bool, bool, bool, bool), Sprite[]> getCollectionCache;
```

Comment:
Rename to something like `getPanelImageCollection` or `getPanelImages` to make it clear what collection it holds?